### ISSUE TITLE
Legitimate domain(s) report - curve.sh

### ISSUE THREAD

**Issue by signalreturn** [MetaMask/eth-phishing-detect#88033]
**Domain:** https://curve.sh/

**Details:** Personal blog on cryptography and crypto in general. Stopped setting up WordPress when I discovered target audience would be served a bright-red warning for the given audience.

curve.sh serves as a domain hack, with the "h" being a C header file (my cope for .s not being a TLD).

Hoping for a technical explanation as to why this new domain was auto-blocked, if my hamming distance to curve.fi is correct, or it was something else. It got blocked before it had ever been shared on social media, so either it was a friends plugin that auto-reported it, or "curve.ccTLD" is just blocked by default.

![image](https://github.com/user-attachments/assets/c1259cf9-5c90-42aa-be9e-5b1aabb4fed1)

Best,
SIGNAL

EDIT: _Additionally, I feel like this blocking might be borderline illegal or anti-competitive in nature, as these lists are sourced from Consensys-sanctioned third parties. **The site does not request a web3.js injection/provider at any point, maybe that should be a guiding heuristic to determine if it should end up there in the first place and not just "is it close to another domain"?**_

**

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
this is very similar to curve.fi which is why it was blocked. 

We don't process unblocking requests for generic blog sites that have no content. This type of domain can be used obviously to phish users.

If you have something stood up here please feel free to open a new issue.

**Comment by signalreturn** [comment]
@AlexHerman1 I'll try to keep my reply civil in tone, but this is the most ridiculous policy I've ever heard and probably illegal. A domain-hack for "curves" has the same root as "curve.fi" so therefore it's blocked by the target audience of the other site.

If that is not sufficient, and it *does* require me to have an actual website finished, please provide an e-mail for Consensys legal, and I'll give this to my counsel to liaise with them about this practice.

You can also DM me this information on Twitter/X: [SIGNAL_RETURN](@SIGNAL_RETURN)

**Comment by signalreturn** [comment]
I'm going to launch a successful DeFi project called Google.fi — Google is now blocked for all Metamask users. 😏
